distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -61,25 +61,22 @@
             <DefaultRolloverStrategy max="10"/>
         </RollingFile>
 
-        <!--TODO DDF-3073
-        The karaf upgrade from 4.0.7 to 4.1.1 introduced a bug with the log4j2 failover
-        appender: "ERROR appender Failover has no parameter that matches element Failovers". See
-        https://issues.apache.org/jira/browse/LOG4J2-1163.-->
-        <!--<RollingFile name="securityBackup" append="true" ignoreExceptions="false"
+        <!-- It is recommended to change the default directory of this log file -->
+        <RollingFile name="securityBackup" append="true" ignoreExceptions="false"
                      fileName="${sys:karaf.data}/log/securityBackup.log"
                      filePattern="${sys:karaf.data}/log/securityBackup.log-%d{yyyy-MM-dd-HH}-%i.log.gz">
             <PatternLayout pattern="[%-5p] %d{ISO8601} | %-16.16t | %-15.20c{1} |  %m%n"/>
             <Policies>
                 <SizeBasedTriggeringPolicy size="20 MB"/>
             </Policies>
             <DefaultRolloverStrategy max="10"/>
-        </RollingFile>-->
+        </RollingFile>
 
-        <!--<Failover name="securityFailover" primary="securityMain">
+        <Failover name="securityFailover" primary="securityMain">
```

Comment:
I think that `<Configuration status="WARN">` at the top of this file should be changed to `<Configuration status="OFF">` or `<Configuration status="FATAL">` to hide the `No Failover Element` found error that shows up on the karaf console on every start-up.